.storybook/stories/playground/dashboard/index.js

```diff
@@ -108,11 +103,9 @@ export const _default = () => {
   };
 
   return (
-    <AppContainer>
-      <Dashboard config={config}>
-        <GlobalStyle />
-        <InterfaceSkeleton />
-      </Dashboard>
-    </AppContainer>
+    <Dashboard config={config}>
```

Comment:
Don't we need to apply styling of dashboard?

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/10601#discussion_r809150060